chsdi/templates/htmlpopup/oev_haltestellen.mako

```diff
@@ -0,0 +1,151 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+    <%!
+        from pyramid.url import route_url
+        import chsdi.lib.helpers as h
+    %>
+    <%
+        protocol = request.scheme
+        lang = request.lang
+        topic = request.matchdict.get('map')
+        host = h.make_agnostic(request.host_url + request.uscript_name)
+
+        id = c['attributes']['nummer']
+        c['baseUrl'] = h.make_agnostic(''.join((protocol, '://', request.registry.settings['geoadminhost'])))
+    %>
+<style>
+  tr:last-child {
+    display:none;
+  }
+</style>
+<script>
+
+var timesRun = 0;
+var dest;
+function getInfos(val){
+    if (val) {
+      dest = val;
```

Comment:
Also I would really rather have the JS part in geadmin. @gjn Too late for that now I guess, but I would really much rather do this properly and avoid polluting the global scope, load this script x times and create another piece of code we cannot test automatically. Enhancement issue?